examples/wifi-echo/server/esp32/main/Kconfig.projbuild

```diff
@@ -35,6 +35,22 @@ menu "WiFi Echo Demo"
             bool "M5Stack"
     endchoice
 
+    choice
+      prompt "Rendezvous Mode"
+      default RENDEZVOUS_MODE_WIFI
+      help
+          Specifies the Rendezvous mode of the peripheral.
+
+      config RENDEZVOUS_MODE_WIFI
+          bool "WIFI"
```

Comment:
Wi-Fi (for consistency with the official Alliance spelling).

Reply:
Done.